src/cloudant/views.py

```diff
@@ -65,6 +89,64 @@ def reduce(self, js_func):
         f = _codify(js_func)
         self[self.view_name]['reduce'] = f
 
+    @property
+    def url(self):
+        """property that builds the view URL"""
+        return posixpath.join(
+            self.design_doc._document_url,
+            '_view',
+            self.view_name
+        )
+
+    def __call__(self, **kwargs):
+        """
+        retrieve data from the view, using the kwargs provided
+        as query parameters
+
+        descending bool
+        endkey string or array
+        endkey_docid  string
+        group bool
+        group_level ??
+        include_docs bool
+        inclusive_end  bool
+        key string
+        limit   int
+        reduce  boolean
+        skip    int
+        stale   enum(ok, update_after)
+        startkey  string or array
+        startkey_docid  string
+
+        """
+        for k in kwargs:
+            if k not in ALL_ARGS:
+                raise ValueError("Invalid argument: {0}".format(k))
+        params = python_to_couch(kwargs)
+        resp = self._r_session.get(self.url, params=params)
+        print ">>>>>",resp.status_code, resp.text
```

Comment:
Probably we should be judicious in our use of print. Prefer loggers and all. In addition, you should use the functional form of print.